docs/api/utility-process.md

```diff
@@ -0,0 +1,128 @@
+# UtilityProcess
+
+`UtilityProcess` creates a child process with
+Node.js and Message ports enabled. It provides the equivalent of [`child_process.fork`][] API from Node.js
+but instead uses [Services API][] from Chromium to launch the child process.
+
+`UtilityProcess` is an [EventEmitter][event-emitter].
+
+## Class: UtilityProcess
+
+> Child process with Node.js integration spawned by Chromium.
+
+Process: [Main](../glossary.md#main-process)<br />
+
+### `new UtilityProcess(modulePath[, args][, options])`
+
+* `modulePath` string - Absolute path to the script that should run as entrypoint in the child process.
+* `args` string[] (optional) - List of string arguments that will be available as `process.argv`
+  in the child process.
+* `options` Object (optional)
+  * `env` Object (optional) - Environment key-value pairs. Default is `process.env`.
```

Comment:
are we missing `cwd`? is that not an option that is available to us? What about the other options in `child_process.fork`?

- `detached` - probably this isn't relevant for this API
- `execPath` - same
- `gid` / `uid`?
- `signal` / `killSignal`?
- `silent`?
- `windowsVerbatimArguments`?
- `timeout`?

Reply:
Let's add at least `cwd`, the other non-patch-requiring things can be implemented by a user app I think.

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/40a545b5da4496080527f11ec348eea7ce703197